src/styles/viewer.styl

```diff
@@ -26,9 +26,18 @@
     flex-grow       1
 
 
-.photo-viewer-nav-previous, .photo-viewer-nav-next
-    flex-shrink      0
-    width            em(75px)
+.pho-viewer-nav-previous, .pho-viewer-nav-next
+    flex-shrink         0
+    width               em(75px)
+    background-position center
+    background-repeat   no-repeat
+    cursor              pointer
+
+.pho-viewer-nav-next
+    background-image embedurl("../assets/icons/icon-arrow-right.svg")
```

Comment:
Can you use single quotes instead of double quotes? The linter (not yet setup on Photo v3 but still) requires single quotes for path and such. 😄